src/Google.Datastore.V1Beta3/DatastoreQueryResults.cs

```diff
@@ -74,22 +68,13 @@ public DatastoreQueryResults(IEnumerable<RunQueryResponse> responses)
             });
 
         /// <summary>
-        /// Returns the results of this query as a sequence of <see cref="RunQueryResponse"/> values
-        /// exactly as returned by the Datastore API. This method is only useful if the application
-        /// wishes to examine the <see cref="RunQueryResponse.Query"/> property; otherwise, use
-        /// <see cref="AsBatches"/> to obtain the sequence of batches.
+        /// This method is for advanced use cases only, where more diagnostic information is required;
```

Comment:
The phrase "advanced use cases" still jumps out at me, and my ego tells me this is the function I want to use, because I'm an advanced user.

I'd rephrase:
This method is for diagnosing underlying RPC calls only.